src/queue_pair.rs

```diff
@@ -274,7 +277,7 @@ impl QueuePair {
     }
 
     /// submit a send request
-    fn submit_send(&self, lms: &[&LocalMemoryRegion], wr_id: WorkRequestId) -> io::Result<()> {
+    fn submit_send(&self, lms: &[&dyn LocalMrReadAccess], wr_id: WorkRequestId) -> io::Result<()> {
```

Comment:
static generic?

Reply:
so many `dyn` in this project, and I will cahnge them to static generic in another commit.